config/webpack/utils/loaders.js

```diff
@@ -51,34 +51,30 @@ const makeCssLoaders = (options = {}) => {
     ...(!server ? [MiniCssExtractPlugin.loader] : []),
     ...cssModuleToTypeScriptLoader,
     {
-      // On the server, we use 'css-loader/locals' to avoid generating a CSS file.
-      // Only the client build should generate CSS files.
-      loader: require.resolve(`css-loader${server ? '/locals' : ''}`),
+      loader: require.resolve('css-loader'),
       options: {
-        modules: true,
+        modules: 'local',
         localIdentName: `${debugIdent}[hash:base64:7]`,
-        minimize: isProductionBuild,
+
+        // On the server, avoid generating a CSS file with exportOnlyLocals.
+        // Only the client build should generate CSS files.
+        exportOnlyLocals: Boolean(server),
         importLoaders: 3
       }
     },
     {
       loader: require.resolve('postcss-loader'),
       options: {
-        plugins: () => [require('autoprefixer')(supportedBrowsers)]
+        plugins: () => [
+          require('autoprefixer')(supportedBrowsers),
+          // Minimize CSS on production builds
+          ...(isProductionBuild ? [require('cssnano')] : [])
```

Comment:
The `postcss-loader` documentation indicates that you need to invoke the function returned from `cssnano`, eg. `require('cssnano')()`, not true? See https://github.com/postcss/postcss-loader#plugins

Reply:
Good call. Fixed.